python/ray/serve/tests/test_cancellation.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, handle):
 
             @app.get("/")
             async def wait_for_cancellation(self):
-                await self._handle.remote()._to_object_ref()
+                _ = self._handle.remote()
```

Comment:
Small nit: why assign it to `_` instead of making the call and not assigning to anything?

Reply:
hmm I don't remember. I think a linter yelled at me